firmware/Makefile

```diff
@@ -6,6 +6,10 @@ default: build
 clean: 
 	rm -rf $(APPLICATION_ROOT)/$(OUTPUT)
 
+nucleo:
```

Comment:
build_nucleo and build_disco would be better I think.

Reply:
or just nucleo and disco

Reply:
See 0c10423 - changed build to disco

Reply:
nice